lib/endpoints/class-wp-rest-users-controller.php

```diff
@@ -235,11 +236,12 @@ public function create_item( $request ) {
 		$this->update_additional_fields_for_object( $user, $request );
 
 		/**
-		 * Fires after a user is created via the REST API
+		 * Fires after a user is created via the REST API.
 		 *
-		 * @param object $user Data used to create user (not a WP_User object)
+		 * @param object          $user    Data used to create the user (not a WP_User object).
 		 * @param WP_REST_Request $request Request object.
-		 * @param bool $bool A boolean that is false.
+		 * @param bool            $bool    A boolean that is false.
+		 * @todo remove final bool = false variable, unused?
 		 */
 		do_action( 'rest_insert_user', $user, $request, false );
```

Comment:
@adamsilverstein The third param here should be `true` on "Create" (it should be false on update).